Packs/RecordedFutureV3/Integrations/RecordedFutureAlerts/RecordedFutureAlerts.py

```diff
@@ -0,0 +1,354 @@
+            params={
+                "alert_type": alert_type,
+                "alert_id": alert_id,
+                "image_id": image_id,
+            },
+            timeout=90,
+            resp_type="content",
+        )
+        return response_content
+
+    def fetch_incidents(self) -> dict[str, Any]:
+        """Fetch incidents."""
+        return self._post(
+            url_suffix="/v3/alert/fetch",
+            json_data={
+                "integration_config": demisto.params(),
+                "query_params": demisto.getLastRun(),
+            },
+            timeout=120,
+        )
+
+
+# === === === === === === === === === === === === === === ===
+# === === === === === === ACTIONS === === === === === === ===
+# === === === === === === === === === === === === === === ===
+
+
+class Actions:
+    def __init__(self, rf_client: Client):
+        self.client = rf_client
+
+    def fetch_incidents(self) -> None:
+        response = self.client.fetch_incidents()
+        alerts = response.get("alerts", [])
+        next_query = response.get("next_query", {})
+
+        incidents = [
+            {
+                "name": alert.get("title"),
+                "occurred": alert.get("created"),
+                "dbotMirrorId": alert.get("id"),
+                "rawJSON": json.dumps(alert),
+            }
+            for alert in alerts
+        ]
+
+        demisto.incidents(incidents)
+        demisto.setLastRun(next_query)
+
+        # update_alert_status = response.pop("alerts_update_data", None)
+        # if update_alert_status:
+        #    self.client.alert_set_status(update_alert_status)
+
+    def get_alerts_command(self) -> dict[str, Any]:
+        """Get Alerts Command."""
+        return self.client.get_alerts()
+
+    def get_alert_rules_command(self) -> dict[str, Any]:
+        """Get Alert Rules Command."""
+        return self.client.get_alert_rules()
+
+    @staticmethod
+    def _get_file_name_from_image_id(image_id: str) -> str:
+        return f"{image_id.replace('img:', '')}.png"
+
+    def _get_image_and_create_attachment(self, alert_type: str, alert_id: str, image_id: str) -> dict | None:
+        try:
+            return_results(f"Trying to fetch {image_id=}")
+            image_content = self.client.get_alert_image(alert_type, alert_id, image_id)
+            file_name = self._get_file_name_from_image_id(image_id)
+            file_result_obj = fileResult(file_name, image_content)
+            demisto.results(file_result_obj)  # Important
+            attachment = {
+                "description": "Alert image",
+                "name": file_result_obj.get("File"),
+                "path": file_result_obj.get("FileID"),
+                "showMediaFile": True,
+            }
+            return attachment
+        except Exception as e:
+            demisto.error(f"Failed to fetch image {image_id}: {str(e)}")
+            return None
+
+    def get_alert_images_command(self) -> list:
```

Comment:
Would be good to document the solution we landed in on a high level, e.g.
```suggestion
    def get_alert_images_command(self) -> list:
    """Fetches images for the current incident, adding them as attachments.
         The attachments are accessible from code via XXX and can easily be displayed in a layout.
         Will fetch at most 25 images per run, but if you run it multiple times it will fetch the next pages"""
```
(the above is probably incorrect or can be expanded upon, feel free)

Reply:
Are we sure that subsequent invocations of the playbook will properly fetch "page 2", "page 3" etc of images? Aka is it official functionality that we should call out here?